addr/validation.go

```diff
@@ -0,0 +1,63 @@
+package addr
+
+import (
+	bitcoincfg "github.com/bitlum/btcd/chaincfg"
+	"github.com/bitlum/connector/chains/dash"
+	"github.com/go-errors/errors"
+	litecoincfg "github.com/ltcsuite/ltcd/chaincfg"
+)
+
+// Validate validates asset's addr in specified net. Returns error if address is invalid or asset is unknown.
+// For ETH net is meaningless, you can use "".
+func Validate(asset string, net string, addr string) error {
```

Comment:
If I understood correctly than after we imported chains all needed networks we added, and there is no need in this function. Basically we just apply `DecodeAddress` function.

Reply:
ETH is not added. Litecoin has own validation chaincfg. I thing explicit validation is better when implicit.